docs/pages/database-access/faq.mdx

```diff
@@ -3,76 +3,102 @@ title: Database Access FAQ
 description: Frequently asked questions about Teleport Database Access.
 ---
 
-# Database Access FAQ
-
 ## Which database protocols does Teleport Database Access support?
 
-Teleport Database Access currently supports PostgreSQL, MySQL, MariaDB and MongoDB
-protocols.
+Teleport Database Access currently supports the PostgreSQL, MySQL, MariaDB and
+MongoDB protocols.
 
 For PostgreSQL and MySQL, both self-hosted and cloud-hosted versions such as AWS
-RDS, Aurora (except for Serverless version which doesn't support IAM auth),
-Redshift, and GCP Cloud SQL are supported. See available [guides](./guides.mdx)
-for all supported configurations.
+RDS, Aurora (except for the serverless version which doesn't support IAM auth),
```

Comment:
Nit: Aurora Serverless is actually a product name which is why it was capitalized here.